### ISSUE TITLE
Change how we display identifiers

### ISSUE THREAD

**Issue by ManonGros** [gbif/hp-grscicoll#181]
As pointed out in this issue: https://github.com/gbif/hp-grscicoll/issues/158

We have too many identifiers and they can be quite confusing.
On each collection/institution page, I would like to have a "how to cite" box with the preferred identifier or the entry URL (if no preferred identifier is provided).

The other identifiers should either be displayed in a collapsable manner or we could also just consider putting a link to the identifier page of the related registry entry.

@MortenHofft would that work?

**Comment by ManonGros** [comment]
Hi @fmendezh I don't think this issue is closed or am I missing something?

**Comment by MortenHofft** [comment]
Yes. But I would need the template for how to create the citation.
And there are still 2 identifers that should be shown I suspect. The primary and the gbif used key.
